spec/features/view_list_of_authors_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.feature "View list of Authors", type: :feature do
+  scenario "with some authors" do
+    create(:author, name: "Martin Fowler")
+    create(:author, name: "Uncle Bob")
+
+    visit authors_path
+
+    expect(page).to have_author("Martin Fowler")
+    expect(page).to have_author("Uncle Bob")
+  end
+
+  scenario "without authors" do
```

Comment:
Yes, controller and action was well covered above. My idea was to show the scenario when we don't have book, but maybe is a good to keep it only on the layer of views spec (and we already have it)